functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/configuration/SurefireForksConfigurationTest.java

```diff
@@ -66,15 +75,38 @@ private void verifyTestSuiteExecution(String... systemPropertiesPairs){
             .applyAsLocalChanges("Inlined variable in a method", "Adds new unit test", "fixes tests");
 
         // when
-        final List<TestResult> actualTestResults =
-            project
-                .build("config/impl-base")
```

Comment:
I specifically narrowed it to this module, as it takes 2 extra minutes to run this test if we run the whole build... :)